### PR TITLE
several misc minor fixes

### PR DESCRIPTION
**Pull request by ctb**
Fixes:

* add *~ to .gitignore
* make sure lambda is present in output maf file; signal that lastal needs upgrade otherwise.
* chmod +x BUSCO/*.py
* add '-f' to BUSCO run so that in edge cases (where BUSCO directory already exists) it will re-run
* formatting fix in installation docs

### CODE REVIEW

File: dammit/tasks.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def get_lastdb_task(db_fn, db_out_prefix, lastdb_cfg, prot=True):
     exc = which('lastdb')
     params = lastdb_cfg['params']
     if prot:
-        params += ' -p'
+        params += ' -p -w3'
```
Comment (camillescott) [inline dammit/tasks.py]: I've added this to `dammit/.config.json` already, so this should be removed (also, the placement here only applies it for protein databases :))

**Comment by camillescott** [comment]
Good changes, thanks! Just the one note on `-w3` to address and I'll merge

**Comment by ctb** [comment]
All good & ready to merge now.

**Comment by camillescott** [comment]
Awesome, thanks! Seeing as I have no CI, i'm just gonna eyeball it say LGTM :)

**Pull request merged**